bitrise.yml

```diff
@@ -158,6 +160,25 @@ workflows:
             echo "-> BITRISE_PUBLIC_INSTALL_PAGE_URL: ${BITRISE_PUBLIC_INSTALL_PAGE_URL}"
             echo "-> BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP: ${BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP}"
 
+  _zero_length_file_upload:
+    steps:
+    - script:
+        inputs:
+        - content: touch 0_length_file.txt
+    - path::./:
+        title: Zero length file upload test
+        run_if: true
+        inputs:
+        - build_url: $BITRISE_BUILD_URL
+        - build_api_token: $BITRISE_BUILD_API_TOKEN
+        - is_compress: "true"
```

Comment:
Does is_compress affect file_length in any way?

Reply:
pr updated: removed options not affecting this test